client/my-sites/themes/single-site-jetpack.jsx

```diff
@@ -79,13 +82,15 @@ const ConnectedSingleSiteJetpack = connectOptions(
 			<div>
 				<SidebarNavigation />
 				<CurrentTheme siteId={ siteId } />
-				<ThemeShowcase { ...props } siteId={ siteId }>
+				<ThemeShowcase { ...props }
+					siteId={ siteId }
+					emptyContent={ showWpcomThemesList ? <div /> : null } >
```

Comment:
Yes, yes. Two things are required to have two lists. Jetpack capabilities are not enough also upload enabled is important.